packages/theme/account.ts

```diff
@@ -35,7 +35,8 @@ export default StyleSheet.create({
   },
 
   text: {
-    ...medium
+    ...medium,
+    ...bold
```

Comment:
This text is used in many places - create a separate `header` style in this stylesheet that has `medium` and `bold`